tamoc/bent_plume_model.py

```diff
@@ -2868,7 +2870,8 @@ def plot_state_space(t, q, q_local, profile, p, particles, fig):
     ax2.set_xlabel('y (m)')
     ax2.set_ylabel('Depth (m)')
     ax2.invert_yaxis()
-    ax2.grid(b=True, which='major', color='0.65', linestyle='-')
+    # b no longer a valid keyword -- what replaced it???
```

Comment:
`b` is no longer an allowed keyword -- but I coulnd't figure out what it did, so I simply removed it -- but there may be a better solution.